showcase/ruby/showcase_integration_test.rb

```diff
@@ -0,0 +1,47 @@
+require "minitest/autorun"
```

Comment:
Maybe have someone else review the ruby files? (If it hasn't been reviewed already)